apps/web/test/validators-comments.test.ts

```
@@ -0,0 +1,493 @@
+// Mock Supabase client
+const mockSupabase = {
+	from: mock(() => mockSupabase),
+	select: mock(() => mockSupabase),
+	eq: mock(() => mockSupabase),
+	single: mock(),
+}
+
+describe('Comment Validation', () => {
+	beforeEach(() => {
+		// Reset all mocks
+		Object.values(mockSupabase).forEach((fn: any) => {
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)